src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java

```diff
@@ -88,9 +88,9 @@ public void onFailure(Throwable throwable) {
 			public void onSuccess(Boolean success) {
 				Mat.hideLoadingMessage();
 				if (success) {
-					view.displaySuccessMessage("Measure is being processed and transferred to MADIE, You will receive an email as soon as transfer is completed");
+					view.displaySuccessMessage("Measure is being processed and transferred to MADiE. A message will be spent to the e-mail associated with this account once the transfer has completed.");
```

Comment:
```suggestion
					view.displaySuccessMessage("Measure is being processed and transferred to MADiE. A message will be sent to the e-mail associated with this account once the transfer has completed.");
```

My bad. Suggested `spent` instead of `sent`.